library/std/src/io/mod.rs

```diff
@@ -2462,6 +2470,17 @@ impl<R: Read> Iterator for Bytes<R> {
             };
         }
     }
+
+    default fn size_hint(&self) -> (usize, Option<usize>) {
+        (0, None)
+    }
+}
+
+#[stable(feature = "bufreader_size_hint", since = "1.51.0")]
```

Comment:
I wasn't sure which version to put, but I marked it as stable due to [this comment](https://github.com/rust-lang/rust/pull/81044#issuecomment-761098908).

Reply:
I think you can just remove it, so long as it stays a private (non-`pub`) trait, since non-exported items don't need stability attributes.  (Tidy will tell you if I'm wrong.)

Reply:
The build failed for the trait implementations if I do that

Reply:
Done